src/lib/db/feature-toggle-client-store.ts

```diff
@@ -176,13 +175,13 @@ export default class FeatureToggleClientStore
         if (!isAdmin) {
             // We should not send strategy IDs from the client API,
             // as this breaks old versions of the Go SDK (at least).
-            this.removeIdsFromStrategies(features);
+            FeatureToggleClientStore.removeIdsFromStrategies(features);
         }
 
         return features;
     }
 
-    private rowToStrategy(row: Record<string, any>): IStrategyConfig {
+    private static rowToStrategy(row: Record<string, any>): StrategySchema {
```

Comment:
Should `IStrategyConfig`/`StrategySchema` be reverted as well?

Reply:
yes I missed that